packages/app-defaults/src/defaults/icons.tsx

```diff
@@ -58,6 +59,7 @@ export const icons = {
   'kind:system': MuiCategoryIcon as IconComponent,
   'kind:user': MuiPersonIcon as IconComponent,
   'kind:resource': MuiWorkIcon as IconComponent,
+  'kind:template': MuiFeaturedPlayListIcon as IconComponent,
```

Comment:
This is obviously a subjective pick of an Icon. Not sure if we have a designer to take a look & review. Otherwise it just might be fine as all the other icons were pretty subjective picks as well.

Reply:
alright - fine for now

Reply:
Shoot, missed this. The Launch Template button looks like this: 
![image](https://github.com/backstage/backstage/assets/67169551/3b1df236-dafd-4ea9-b84c-8a778bf906c1)

But the icon selected here looks like this:
![image](https://github.com/backstage/backstage/assets/67169551/9169adbe-82ba-4e0c-a44c-6576a99d753e)

Should these be these same? Clearly for a follow up PR